_layouts/styleguide.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
-<html lang="en">
+<!--[if lt IE 9]><html class="lt-ie9"><![endif]-->
```

Comment:
It's not in this PR but it's in our styleguide.css - we have the class used for the sidebar bc we need a different approach for IE.